libs/Microsoft.MixedReality.WebRTC/PeerConnection.cs

```diff
@@ -1468,12 +1468,6 @@ public void RemoveDataChannel(DataChannel dataChannel)
 
                 var res = PeerConnectionInterop.PeerConnection_RemoveDataChannel(_nativePeerhandle, dataChannel._nativeHandle);
                 Utils.ThrowOnErrorCode(res);
-
-                DataChannelRemoved?.Invoke(dataChannel);
-
-                // PeerConnection is owning the data channel, and all internal states have been
-                // updated and events triggered, so notify the data channel to clean its internal state.
-                dataChannel.DestroyNative();
```

Comment:
I noticed this had some comments about PeerConnection owning this instance, but I get the feeling it is more owned by the peer connection interop, since that is the one that destroys the native implementation, and also actually triggers events that this PeerConnection class only wrapps in a way.

Reply:
Yes. But the interop is kind of internal, so conceptually from the end-user point of view using the public API, the ownership is with the `PeerConnection`. But you're right, that derives from the fact the native data channel is owned by the native peer connection.